include/opcode/riscv-opc.h

```diff
@@ -710,6 +776,7 @@
 #define CSR_MARCHID 0xf12
 #define CSR_MIMPID 0xf13
 #define CSR_MHARTID 0xf14
+#define CSR_MENTROPY 0xf15
```

Comment:
Maybe you forgot this one :)

Reply:
Thank you for your metion, I had update the 'mentorpy' case.